tests/tasks/setup/eks/awscli-mng.yaml

```diff
@@ -77,15 +77,16 @@ spec:
       for i in $(seq 1 $asgs)
       do
         #max number of nodes MNG allows per ASG
-        create_dp $i 1000
+        create_dp $i 1000 c5.large,m5.large,r5.large,t3.large,t3a.large,c5a.large,m5a.large,r5a.large
```

Comment:
Could you parametrize these and default it to these instance types ? 
Given you were running into icing issues when you tried it with different set of instances when testing prom changes  -  It will be much easier to control the mixed-instance-group dynamically.

Reply:
true..have added this.

Reply:
You mean, you are going to add it ?

Reply:
yeah pushed in a different commit.